packages/odf/components/bucket-class/wizard-pages/general-page.tsx

```diff
@@ -110,40 +163,33 @@ const GeneralPage: React.FC<GeneralPageProps> = ({ dispatch, state }) => {
             })}
           </FormGroup>
         )}
-        <FormGroup
-          labelIcon={
-            <FieldLevelHelp>
-              <ul>
-                <li>{t('3-63 chars')}</li>
-                <li>{t('Starts and ends with lowercase number or letter')}</li>
-                <li>
-                  {t(
-                    'Only lowercase letters, numbers, non-consecutive periods or hyphens'
-                  )}
-                </li>
-                <li>{t('Avoid using the form of an IP address')}</li>
-                <li>{t('Globally unique name')}</li>
-              </ul>
-            </FieldLevelHelp>
-          }
-          className="nb-create-bc-step-page-form__element"
-          fieldId="bucketclassname-input"
-          label={t('BucketClass name')}
-          helperText={t(
-            'A unique name for the bucket class within the project.'
-          )}
-        >
-          <TextInput
-            data-test="bucket-class-name"
-            placeholder={t('my-multi-cloud-mirror')}
-            type="text"
-            id="bucketclassname-input"
-            value={state.bucketClassName}
-            onChange={onChange}
-            validated={validated}
-            aria-label={t('BucketClass Name')}
-          />
-        </FormGroup>
+
+        <TextInputWithFieldRequirements
+          control={control}
+          fieldRequirements={fieldRequirements}
+          popoverProps={{
+            headerContent: t('Name requirements'),
+            footerContent: `${t('Example')}: my-multi-cloud-mirror`,
+          }}
+          formGroupProps={{
+            className: 'nb-create-bc-step-page-form__element',
+            fieldId: 'bucketclassname-input',
+            label: t('BucketClass name'),
+            helperText: t(
+              'A unique name for the bucket class within the project.'
```

Comment:
it is a convention: CR names should be mentioned same as they appear in their CR's "kind".
```suggestion
              'A unique name for the BucketClass within the project.'
```

Reply:
done